.codecov.yml

```diff
@@ -0,0 +1,12 @@
+comment: off
+github_checks:
+  annotations: false
+coverage:
+  status:
+    patch:
+      default:
+        target: 95%
+        only_pulls: true
```

Comment:
why do we choose `true`? because we assume everything goes through a PR?

Reply:
This avoids checks in commits pushed directly to the repo. I let this config (from solidity contracts) because, as you say, I assume everything should come through PRs.

Reply:
after speaking with Fran he explained this was to be able to force merge a PR that does not pass coverage without getting a second check. this isn't something we were forced to do yet and not sure if we care about the 2nd check. we could have the check on single commits and see how it goes.